src/js/controller.js

```diff
@@ -28,38 +28,30 @@ export default class Controller extends Component {
         this.confirmationModal = new ConfirmationModal(this, ConfirmationModal.COOKIE_NAMES.DATA_MAPPER_VERSION_SELECTION);
 
         $(window).on('hashchange', () => {
-            // On every hash change the render function is called with the new hash.
-            // This is how the navigation of our app happens.
-            this.api.cancelAndInitAbortController();
```

Comment:
I don't see cancelAndInitAbortController used anywhere else, it can be removed from the API now right?

Reply:
updated @michaelglenister